client/src/GamemasterInGame/CharacterSelectList.tsx

```diff
@@ -94,44 +94,46 @@ export function CharacterSelectList({
 
   return (
     <Flex gap="1" direction="column" px="3">
-      {Object.entries(charactersByType).map(([characterType, characters]) => (
-        <React.Fragment key={characterType}>
-          <Heading
-            size="4"
-            id={characterType}
-            align="right"
-            color={colorMap[characterType as CharacterType]}
-          >
-            {characterType}
-          </Heading>
-          {characters.map(({ id }) => (
-            <Flex
-              gap="1"
-              align={"center"}
-              key={id}
-              style={{ height: "2em" }}
-              asChild
+      {Object.entries(charactersByType)
+        .filter(([_, characters]) => characters.length > 0)
```

Comment:
Formatting must have gotten weird, cause this is the only line that should be changed.